playbooks/tasks/portal-health-check-enable.yml

```diff
@@ -1,8 +1,18 @@
 ---
-
 # Enable portal health check to add portal to load balancer
 
-- name: Enable portal healh check to add portal to load balancer
-  command: docker exec health-check cli/enable
-  become: True
-  become_user: "user"
\ No newline at end of file
+# Check if the health check container is running
+- name: Check health check container is running
+  community.docker.docker_container_info:
+    name: health-check
+  register: health_check_docker_container_result
+  # If the container is not up and running, fail here so that it is clear the
+  # reason for the failure is the docker container and not the cli command.
+  failed_when: >
+    (health_check_docker_container_result.exists is not defined) or
+    (health_check_docker_container_result.container is not defined) or
+    (not health_check_docker_container_result.container.State.Running)
+
+# Mark the health check as enabled via the cli command
+- name: Enable portal health check to add portal to load balancer
+  command: docker exec health-check cli enable
```

Comment:
do we want to make sure that it worked by adding a check whether http://10.10.10.60:3100/health-check returns response json where "disabled" equals "false" ?

this can be a follow up too, I'll approve this just in case anyway